app/src/main/java/com/github/gotify/service/WebSocketService.java

```diff
@@ -1,5 +1,7 @@
 package com.github.gotify.service;
```

Comment:
What's the use-case, I don't think this is needed as you can already open urls, no need to do this separately in background.